### PR TITLE
[compiler][lowering] .parent initialization fix

### PR DESCRIPTION
**Pull request by vvlevchenko**

### CODE REVIEW

File: backend.native/cli.bc/src/org/jetbrains/kotlin/cli/bc/K2NativeCompilerArguments.kt [hunk 1]
```diff
@@ -163,6 +163,9 @@ class K2NativeCompilerArguments : CommonCompilerArguments() {
     @Argument(value = "-Xdebug-info-version", description = "generate debug info of given version (1, 2)")
     var debugInfoFormatVersion: String = "1" /* command line parser doesn't accept kotlin.Int type */
 
+    @Argument(value = "-Xcheck-ir-parents", description = "Don't link the libraries from dist/klib automatically")
```
Comment (olonho) [inline backend.native/cli.bc/src/org/jetbrains/kotlin/cli/bc/K2NativeCompilerArguments.kt]: Incorrect description.

File: buildSrc/plugins/src/main/groovy/org/jetbrains/kotlin/KonanTest.groovy [hunk 2]
```diff
@@ -94,6 +94,10 @@ abstract class KonanTest extends JavaExec {
     abstract void compileTest(List<String> filesToCompile, String exe)
 
     protected void runCompiler(List<String> filesToCompile, String output, List<String> moreArgs) {
+        def extopts = []
+        if (project.hasProperty('checkIrParents') && (project.property('checkIrParents') == 'true')) {
```
Comment (olonho) [inline buildSrc/plugins/src/main/groovy/org/jetbrains/kotlin/KonanTest.groovy]: Can we make it one function?

**Comment by olonho** [comment]
Needs rebase.

**Comment by SvyatoslavScherbina** [comment]
No activity, closing.

**Pull request closed without merging**